delta/sdk/src/main/resources/contexts/projects-metadata.json

```diff
@@ -4,6 +4,18 @@
     "_uuid": "https://bluebrain.github.io/nexus/vocabulary/uuid",
     "_organizationUuid": "https://bluebrain.github.io/nexus/vocabulary/organizationUuid",
     "_organizationLabel": "https://bluebrain.github.io/nexus/vocabulary/organizationLabel",
+    "_effectiveApiMappings": {
+      "@id": "https://bluebrain.github.io/nexus/vocabulary/effectiveApiMappings",
+      "@container": "@set"
+    },
+    "namespace": {
+      "@id": "https://bluebrain.github.io/nexus/vocabulary/namespace",
+      "@type": "@id"
+    },
+    "prefix": {
+      "@id": "https://bluebrain.github.io/nexus/vocabulary/prefix",
```

Comment:
it is metadata, it must start by `_`
The value of prefix is not an id, so `@type: @id` is wrong